### DISCUSSION TITLE
ncorrect encoding in the plugin

### DISCUSSION THREAD

**Discussion by groseal** [gitbito/bitoai#218] [Q&A]
I used the assistant all evening without any problems, but in the morning the same problem arose. I did not turn off the computer at night (it was in sleep mode), the operating system is windows 11, IntelliJ IDEA 2023.1.6 (Ultimate Edition). I don't know how to fix this problem, please help.
![1](https://github.com/user-attachments/assets/2ef8637b-1c4a-4222-a188-335c22ea72f3)

**Comment by MRTIBBETS** [comment]
Hey @groseal — apologies for the slow reply. Is this issue still occurring in your Bito VS Code extension? Can you upgrade to the latest release and try again? Let us know if you still see this issue and our engineering team will jump on it.
